### PR TITLE
Adding face-on/-away tests for mutli_inspiral

### PR DESCRIPTION
**Pull request by a-r-williamson**
These files can be used to run tests of `pycbc_multi_inspiral`, in particular the face-on/-away signal projection.

### CODE REVIEW

**Comment by a-r-williamson** [comment]
@pannarale Boop

File: test/multi_inspiral/bank_veto_bank.xml [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+		<Column Type="lstring" Name="username"/>
+		<Column Type="int_4s" Name="unix_procid"/>
+		<Column Type="int_4s" Name="start_time"/>
+		<Column Type="int_4s" Name="end_time"/>
+		<Column Type="int_4s" Name="jobid"/>
+		<Column Type="lstring" Name="domain"/>
+		<Column Type="lstring" Name="ifos"/>
+		<Column Type="int_8s" Name="process_id"/>
+		<Stream Name="processgroup:process:table" Type="Local" Delimiter=",">
+			"tmpltbank","645a41a2fa3df783221bd9988f70b0cee1861d56","UNCLEAN: some modifications were not commited!",946819435," ",0,"titan1.atlas.aei.uni-hannover.de","spxiwh",16799,956049146,956049411,0,"lalapps","H1",0
```
Comment (pannarale) [inline test/multi_inspiral/bank_veto_bank.xml]: The comment "UNCLEAN [...]" can be removed; it's from cvs days and would not look good on master

**Review by pannarale — CHANGES_REQUESTED** [review]
All 3 files look sane.  Just a minor change is needed for the xml file.

File: test/multi_inspiral/bank_veto_bank.xml (hunk 1 above)
Comment (spxiwh) [reply test/multi_inspiral/bank_veto_bank.xml]: This caught my eye! This file dates from:
```
(pycbc_test) [iwharry@harrymac relbin]$ lalapps_tconvert 946819435
Wed Jan 06 13:23:40 UTC 2010
```

Might be time for a new bank_veto bank ;)

Comment (a-r-williamson) [reply test/multi_inspiral/bank_veto_bank.xml]: Thanks, @pannarale I've removed that now.

@spxiwh Yes, a new veto bank for O4 is on the to-do list!

**Review by pannarale — APPROVED** [review]

**Pull request merged**